VirtwhoQE/virtwho-ci
# Patch rhel7 7 new feature

New add 3 testcases:
1. tc_1108_check_hypervisor_facts.py
      Update virt_who/testing.py, get "dmi" and "cluster" value from facts.
      Test steps are:
      - run virt-who to get facts info
      - check facts containing all necessary keys and values
      - check hypervisor.type value right
      - check dmi.system.uuid value right
      - check hypervisor.cluster value right (only for rhevm and esx)

2.tc_2060_check_commented_out_line_with_tab_space.py
    Test steps are:
     -  run virt-who with all good configurations
     -  add useless line "xxx=xxx" with tab space below "server=" option -> the bad line makes server value wrong and failed to report mapping.
     - commnet out the useless line "#xxx=xxx" -> virt-who report ok

3.tc_2061_check_fake_mode_is_hypervisor_option.py
    Test steps are:
    - run remote hypervisor with is_hypervisor=False -> failed to start virt-who
    - run local hypervisor with `is_hypervisor=True` -> failed to start virt-who
    - disable is_hypervisor= option -> the default is_hypervisor=True will be used
